internal/httpclient/models/default_session.go

```diff
@@ -0,0 +1,172 @@
+// Code generated by go-swagger; DO NOT EDIT.
+
+package models
+
+// This file was generated by the swagger tool.
+// Editing this file might prove futile when you re-run the swagger generate command
+
+import (
+	"context"
+
+	"github.com/go-openapi/errors"
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+	"github.com/go-openapi/validate"
+)
+
+// DefaultSession IDTokenSession is a session container for the id token
+//
+// swagger:model DefaultSession
+type DefaultSession struct {
+
+	// claims
+	Claims *IDTokenClaims `json:"Claims,omitempty"`
```

Comment:
These are generated and do not follow the same naming convention as other mappings (snake_case)

Reply:
This can be fixed by adding the json tags in fosite

https://github.com/ory/fosite/blob/37da2286949fd69106fe72569fb430583032639e/session.go#L54

Please keep in mind that fosite on master does not work with v1.x since it is already on the v2.x branch. That means changes have to be made against this commit

5dbfa9a56d36061d5bf80149e1801c36a371bafd

and then PRed into this branch:

https://github.com/ory/fosite/commits/v0.42.x

and then also be made against master.

Alternatively, we could copy&paste the default session object into hydra, and use that struct isntead of the struct from fosite across the code base. Given the complexity of the git operations in fosite, it might be an easier solution.

Reply:
Sorry, this is the correct type: https://github.com/ory/fosite/blob/d71c67ff5431c0ce7a47cda6b871d4fd68e7247e/handler/openid/strategy_jwt.go#L53

Reply:
For master - https://github.com/ory/fosite/pull/688

Reply:
👍 merged :)

Reply:
So, how do I bump `fosite` to the updated version? I see that in `go.mod` it is pointing to some commit `github.com/ory/fosite v0.42.3-0.20220728130632-c3e0133cfed9`
The middle part is a bit tricky for me to figure out - is it a timestamp?

Reply:
replace it with the git hash and run go mod tidy after :)